src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -85,6 +88,9 @@
 public final class ValidateVariants extends VariantWalker {
     static final Logger logger = LogManager.getLogger(ValidateVariants.class);
 
+    public static final String validateGVCF = "validateGVCF";
```

Comment:
@jsotobroad Our convention is that static constants are UPPERCASE_WITH_UNDERSCORES.